docs/content/integrations/embedded-elt.mdx

```diff
@@ -0,0 +1,126 @@
+---
+title: "Dagster Embedded ELT"
+description: Lightweight ELT with Dagster
+---
+
+# Dagster Embedded ELT
+
+This package provides a framework for building ELT pipelines with Dagster through helpful pre-built assets and resources. It is currently in experimental development, and we'd love to hear your feedback.
+
+This package currently includes a single implementation using <a href="https://slingdata.io">Sling</a> provides a simple way to sync data between databases and file systems.
+
+We plan on adding additional embedded ELT tool integrations in the future.
+
+## Relevant APIs
+
+| Name                                                                        | Description                                                                    |
+| --------------------------------------------------------------------------- | ------------------------------------------------------------------------------ |
+| <PyObject module="dagster_embedded_elt.sling" object="build_sling_asset" /> | The core Sling asset factory for building syncs                                |
+| <PyObject module="dagster_embedded_elt.sling" object="SlingResource" />     | The Sling Resource used for handing credentials to databases and object stores |
+
+---
+
+## Getting Started
+
+To get started with `dagster-embedded-elt` and Sling, first familiarize yourself with <a href="https://docs.slingdata.io/sling-cli/running-tasks">Sling's documentation</a>
+
+First, setup a <PyObject module="dagster-embedded-elt.sling" object="SlingResource" /> which defines the source and destination credentials. The <PyObject module="dagster-embedded-elt.sling" object="SlingResource" /> requires both a <PyObject module="dagster-embedded-elt.sling" object="SlingSourceConnection" /> and a <PyObject module="dagster-embedded-elt.sling" object="SlingTargetConnection" />.
+
+You can provide either a connection string or a dictionary of connection parameters to each of these classes.
+
+For details on what connection parameters Sling accepts for each integration, see the <a href="https://docs.slingdata.io/connections/database-connections">Sling Connections</a> page.
+
+To create an Asset that syncs between two connections, you can use the <PyObject module="dagster-embedded-elt.sling" object="build_sling_asset" /> factory.
+
+---
+
+## Setting up a Sling Resource
+
+A Sling Resource is a Dagster Resource that contains references to both a Source Connection and a Target Connection. Sling is versatile in what a source or destination can represent. You can provide arbitrary keywords to the <PyObject module="dagster-embedded-elt.sling" object="SlingSourceConnection" /> and <PyObject module="dagster-embedded-elt.sling" object="SlingTargetConnection" /> classes.
```

Comment:
```suggestion
A Sling resource is a Dagster resource that contains references to both a Source Connection and a Target Connection. Sling is versatile in what a source or destination can represent. You can provide arbitrary keywords to the <PyObject module="dagster-embedded-elt.sling" object="SlingSourceConnection" /> and <PyObject module="dagster-embedded-elt.sling" object="SlingTargetConnection" /> classes.
```

`resource` doesn't need to be capitalized - it's not a proprietary term for us.

I'm also leaning towards `Source` and `Target Connection` not needing to be capitalized either. Sling uses these terms and they're not capitalized in their docs, so I think we should be consistent with their usage since those seem to be their terms.